src/adagrams.js

```diff
@@ -1,15 +1,157 @@
+// import Math from 'core-js/library/fn/math';
```

Comment:
Accidental import? Generally, it's a good idea to remove commented out code before submitting a PR!